composer.json

```diff
@@ -23,16 +23,40 @@
     "mezzio-hal"
   ],
   "require": {
-    "php": "~7.4.0 || ~8.0.0 || ~8.1.0",
-    "mezzio/mezzio-hal": "^2.4.0"
+    "php": "~8.1.0 || ~8.2.0",
+    "mezzio/mezzio-hal": "^2.4.0",
+    "doctrine/common": "^3.4"
   },
   "require-dev": {
-    "phpunit/phpunit": "^9.5.20",
-    "squizlabs/php_codesniffer": "^3.6.2"
+    "phpunit/phpunit": "^10.2",
+    "laminas/laminas-coding-standard": "^2.5",
+    "vimeo/psalm": "^5.13"
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "DotTest\\DoctrineMetadata\\": "test/"
+    }
   },
   "autoload": {
     "psr-4": {
       "Dot\\DoctrineMetadata\\": "src"
```

Comment:
The last 'last thing' 😄 
Please add missing slash after the `src` directory as specified in the [Composer documentation](https://getcomposer.org/doc/01-basic-usage.md#autoloading).